tools/flow9/backends/type_hierarchy.flow

```diff
@@ -17,6 +17,9 @@ export {
 	);
 
 	extractPolymorphism(acc : Set<BTypePar>, b : BType) -> Set<BTypePar>;
+
+	// find a match between polyType (BTypePar) and real type (BTypeName)
```

Comment:
Add an example to clarify. I guess it is Pair<?, ??> against Pair<int, string> and you get ? -> int, ?? -> string?